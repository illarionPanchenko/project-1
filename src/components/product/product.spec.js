import React from 'react';
import {Provider} from "react-redux";
import { shallow, configure  } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Product from "./product";
import { store } from '../../index';

configure({ adapter: new Adapter() });

const enzymeWrapper = shallow(<Provider store={store}><Product /></Provider>);

describe('Product',()=>{
    it('snapshot', () => {
        expect(enzymeWrapper).toMatchSnapshot();
    })
});