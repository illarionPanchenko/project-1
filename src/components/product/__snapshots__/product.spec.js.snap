// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product snapshot 1`] = `ShallowWrapper {}`;
